Sources/DateTime/InternalUTCClock.swift

```diff
@@ -30,6 +30,10 @@ public struct InternalUTCClock {
             self._value = _value
         }
 
+        public init(secondsComponent: Int64, attosecondsComponent: Int64) {
```

Comment:
Maybe remove ```Components```? 
```
let instant = InternalUTCClock.Instant(secondsComponent: timestamp.seconds, attosecondsComponent: timestamp.attoseconds)
```

Reply:
It is matching `Swift.Duration` in naming convention, thus `Components`...